src/test/java/org/opentripplanner/updater/trip/moduletests/delay/SkippedTest.java

```diff
@@ -6,42 +6,43 @@
 import static org.junit.jupiter.api.Assertions.assertNotSame;
 import static org.junit.jupiter.api.Assertions.assertNull;
 import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.opentripplanner.transit.model._data.TransitModelForTest.id;
 import static org.opentripplanner.updater.spi.UpdateResultAssertions.assertSuccess;
-import static org.opentripplanner.updater.trip.RealtimeTestEnvironment.SERVICE_DATE;
 import static org.opentripplanner.updater.trip.UpdateIncrementality.DIFFERENTIAL;
 
 import org.junit.jupiter.api.Test;
 import org.opentripplanner.transit.model.framework.FeedScopedId;
 import org.opentripplanner.transit.model.timetable.RealTimeState;
 import org.opentripplanner.transit.model.timetable.TripTimesStringBuilder;
+import org.opentripplanner.updater.trip.RealtimeTestConstants;
 import org.opentripplanner.updater.trip.RealtimeTestEnvironment;
 import org.opentripplanner.updater.trip.TripUpdateBuilder;
 
 /**
  * A mixture of delayed and skipped stops should result in both delayed and cancelled stops.
  */
-public class SkippedTest {
+class SkippedTest implements RealtimeTestConstants {
 
   @Test
   void scheduledTripWithSkippedAndScheduled() {
-    var env = RealtimeTestEnvironment.gtfs();
-    String scheduledTripId = env.trip2.getId().getId();
+    var env = RealtimeTestEnvironment.gtfs().withTrip2().build();
 
-    var tripUpdate = new TripUpdateBuilder(scheduledTripId, SERVICE_DATE, SCHEDULED, env.timeZone)
+    var tripUpdate = new TripUpdateBuilder(TRIP_2_ID, SERVICE_DATE, SCHEDULED, TIME_ZONE)
       .addDelayedStopTime(0, 0)
       .addSkippedStop(1)
       .addDelayedStopTime(2, 90)
       .build();
 
     assertSuccess(env.applyTripUpdate(tripUpdate));
 
-    assertOriginalTripPatternIsDeleted(env, env.trip2.getId());
+    var trip2Id = id(TRIP_2_ID);
+    assertOriginalTripPatternIsDeleted(env, trip2Id);
 
-    assertNewTripTimesIsUpdated(env, env.trip2.getId());
+    assertNewTripTimesIsUpdated(env, trip2Id);
```

Comment:
Just a nitpick but you could make the assertXxx() methods take the id as a string instead and you wouldn't need the variable. My gut feeling is that you would save a couple of lines in this file that way :)

Reply:
I reworked the handling of the IDs here: c97d90a2bff2948d4a62e253fc1fe9fb0b2a6dd6